launch/ur10e_bringup.launch

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+  Universal robot ur10e launch.  Loads ur10e robot description (see ur_common.launch
+  for more info)
+
+  Usage:
+    ur10e_bringup.launch robot_ip:=<value>
+-->
+<launch>
+
+  <!-- robot_ip: IP-address of the robot's socket-messaging server -->
+  <arg name="robot_ip"/>
+  <arg name="limited" default="false"/>
+  <arg name="min_payload"  default="0.0"/>
+  <arg name="max_payload"  default="10.0"/>
+  <arg name="prefix" default="" />
+  <arg name="use_lowbandwidth_trajectory_follower" default="false"/>
+  <arg name="time_interval" default="0.008"/>
+  <arg name="servoj_time" default="0.008" />
+  <arg name="servoj_time_waiting" default="0.001" />
```

Comment:
I remember not being able to run this launch file out of the box due to these args existing. Can you verify?